libs/ui/src/prose.tsx

```diff
@@ -95,3 +97,7 @@ export const Prose = styled('div')<ProseProps>`
     border-top: 0.1em solid #666666;
   }
 `;
+
+export function Prose(props: ProseProps): JSX.Element {
+  return <ProseContainer {...props} />;
+}
```

Comment:
Could you elaborate more about your comment in the PR description about switching these to `React` components? Specifically "That's preferable anyway, to keep the shared components opinionated and standardized."

Reply:
Sure thing - a `styled.div` will expose all possible props for `<div>`, whereas if we only export the wrapper component, we can limit what props are accepted to just what's relevant for rendering a piece of prose text, reducing and simplifying the API surface area.

That also makes sure that if we want to add something like a a11y props, we have to explicitly add them within this component (or expose additional props that control which a11y props are passed to the div) rather than allowing one-off a11y settings to be added in one frontend and not the other (if that makes sense)